CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.8.9] - 2023-06-21
```

Comment:
Adding a new feature should bump the minor version number, not the patch number. 
But in this case, unless I am misunderstanding the change to the modify_config_map function, the change being made breaks backwards compatibility (specifically, an additional argument is now required when calling the **modify_config_map** function). That being the case, it would be appropriate to bump the major version number.

That can be avoided by making the new argument optional (by providing a default value that results in the current behavior) and making it the final argument (so existing calls to the function will still work).